### PR TITLE
Created Components for each section of Order Details

### LINKED ISSUE #105: Create a component for each section of `OrderDetails`

### PR DESCRIPTION
**Pull request by BoJoNVi**
Fixes #105 Each section component is made inside the folder  ```Components/OrderDetails```.

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] screenshots/GIFs are attached :paperclip: in case of UI changes

### CODE REVIEW

**Comment by BoJoNVi** [comment]
@jcads this is ready to be reviewed. :D

File: src/pages/orders/OrderDetails.js [hunk 1]
```diff
@@ -13,27 +13,14 @@ import {
   Button,
 } from "reactstrap";
 
-import DateFormat from "../../utils/dateFormat";
 import BackButton from "../../components/BackButton/BackButton";
+import OrderDetailsOrder from "../../components/OrderDetails/OrderDetailsOrder";
+import OrderDetailsBuyer from "../../components/OrderDetails/OrderDetailsBuyer";
+import OrderDetailsStatus from "../../components/OrderDetails/OrderDetailsStatus";
```
Comment (jcads) [inline src/pages/orders/OrderDetails.js:20]: Just put the components inside the orders directory as these components are not shared.

**Review by jcads — COMMENTED** [review]
Looks good! I just have a small suggestion. See inline comments

File: src/pages/orders/OrderDetails.js [hunk 2]
```diff
@@ -13,27 +13,14 @@ import {
   Button,
 } from "reactstrap";
 
-import DateFormat from "../../utils/dateFormat";
 import BackButton from "../../components/BackButton/BackButton";
+import OrderDetailsBuyer from "./OrderDetailsBuyer";
+import OrderDetailsOrder from "./OrderDetailsOrder";
+import OrderDetailsStatus from "./OrderDetailsStatus";
```
Comment (jcads) [inline src/pages/orders/OrderDetails.js:19]: Btw, good naming convention :+1:

File: src/pages/orders/OrderDetails.js [hunk 3]
```diff
@@ -60,137 +47,13 @@ const OrderDetails = (props) => {
...
-              {order.instruction ? (
-                <div className="d-sm-flex flex-wrap">
-                  <h3 className="mb-0 mr-2">Special Instructions: </h3>
-                  <label>{order.instruction}</label>
-                </div>
-              ) : null}
-            </div>
+            <OrderDetailsBuyer order={order} />
             <hr />
             {/* Order Details */}
```
Comment (jcads) [inline src/pages/orders/OrderDetails.js:52]: remove comment

File: src/pages/orders/OrderDetails.js [hunk 4]
```diff
@@ -60,137 +47,13 @@ const OrderDetails = (props) => {
...
-                        <label>Php {itemTotal}</label>
-                      </div>
-                    </Col>
-                  </Row>
-                </div>
-              );
-            })}
+            <OrderDetailsOrder order={order} />
             <hr />
             {/* Order Status */}
```
Comment (jcads) [inline src/pages/orders/OrderDetails.js:55]: remove comment

File: src/pages/orders/OrderDetails.js (hunk 2 above)
Comment (jcads) [inline src/pages/orders/OrderDetails.js:19]: Btw, good naming convention :+1:

**Review by jcads — APPROVED** [review]
Lgtm :+1:
Just remove the comments as it is redundant because we can now see what component it is based on component's name.

**Pull request merged**